src/shared/containers/WorkSpaceList.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import { useAsyncEffect } from 'use-async-effect';
+import { Resource } from '@bbp/nexus-sdk';
+import { useNexusContext } from '@bbp/react-nexus';
+import TabList from '../components/Tabs/TabList';
+import DashBoardList from './DashBoardList';
+
+interface WorkSpaceListProps {
+  workSpaceIds: string[];
+  orgLabel: string;
+  projectLabel: string;
+}
+
+const WorkSpaceList: React.FunctionComponent<WorkSpaceListProps> = ({
+  workSpaceIds,
+  orgLabel,
+  projectLabel,
+}) => {
+  const [workSpaces, setWorkSpaces] = React.useState<Resource[]>([]);
+  const [selectedWorkSpace, setSelectedWorkSpace] = React.useState<Resource>();
+  const nexus = useNexusContext();
+  const selectWorkSpace = (id: string) => {
+    const w = workSpaces.find(w => w['@id'] === id);
+    setSelectedWorkSpace(w);
+  };
+  useAsyncEffect(async () => {
+    const workSpaceList: Resource[] = [];
+    for (let i = 0; i < workSpaceIds.length; i +=1) {
+      const id: string[] = workSpaceIds[i].split('/');
```

Comment:
This is very risky... You rely on the the fact the last bit of the `@id` value is the `resourceId` which I don't think is always the case... Plus there is exception handling in case that resource doesn't resolve.

Why not simply use the `outgoing` API to get a list of _children_ resources, and only keep the ones that are type `Dashboard`? Like we do in nexus web, here for example http://localhost:8000/bbp-test/studio/resources/https%3A%2F%2Fdev.nexus.ocp.bbp.epfl.ch%2Fv1%2Fresources%2Fbbp-test%2Fstudio%2F_%2F4c32700e-a558-4015-badc-4f79cdfcc1bb?rev=1#links